### ISSUE TITLE
Preparation work for 4.33(2024-09) and open master for development

### ISSUE THREAD

**Issue by MohananRahul** [eclipse-platform/eclipse.platform.releng.aggregator#2056]
This preparation work involves the following tasks. For previous issue please refer to #1841 

- [ ] Create R4_32_maintenance branch
- [ ] Update R4_32 maintenance branch with release version for 4.32+ changes
- [ ] Move 4.32-I builds to R4_32_maintenance branch
- [ ] Update Parent pom and target sdk deployment jobs for R4_32_maintenance branch
- [ ] Create new test jobs for 4.33
- [ ] Configure SWT build scripts for 4.33
- [ ] Splash Screen for 4.33 (2024-09)
- [ ] Create 4.33-I-builds repo
- [ ] POM and product version changes for 4.33 release
- [ ] Update product version number to 4.33 across build scripts
- [ ] Move previous version to 4.32RC2 across build scripts
- [ ] Update version number in Mac's Eclipse.app for 4.33
- [ ] Clean forceQualifierUpdate files for doc bundles
- [ ] Cleanup approved api list
- [ ] Update builds and repo cleanup scripts for 4.33
- [ ] Create new I-build job for 4.33 release
- [ ] Update check composites script to verify 4.33 repositories
- [ ] Update Comparator repo and eclipse run repo to 4.33-I-builds repo
- [ ] Version bumps for 4.33 stream 
- [ ] Update previous release version to 4.32 GA across build scripts 
- [ ] Update build calendar for platform 4.33 release
- [ ] Deploy ecj compiler from 4.32 GA and use it for 4.33 M1 build
- [ ] Update generic repos I-builds, Y-builds, P-builds to point to 4.33 repos

**Comment by merks** [comment]
We should avoid to forget to update the following for this release cycle:

![image](https://github.com/eclipse-platform/eclipse.platform.releng.aggregator/assets/208716/506aec8e-a510-41b1-9003-2f8bfafcea42)

That makes me wonder about perhaps using this instead:

https://download.eclipse.org/releases/latest/

Of course we can't use

https://download.eclipse.org/releases/2024-09/

until it exists after the release.

But it seems strange and somehow inconsistent to refer to the Platform's coming (and empty) release repository while also referring to SimRel's ongoing release repository which contains milestones. 

This then also begs the question, why not use this:

https://download.eclipse.org/eclipse/updates/latest

If both just reference a latest composite that changes when there is a release, there is no longer any maintenance effort, the reference will always be the correct one, and the users preferences don't fill up slowly with stale old release repositories..

/cc @jonahgraham @akurtakov 

Do you guys have thoughts and opinions about this?

**Comment by laeubi** [comment]
Maybe its worth to even use https://download.eclipse.org/releases/milestone/ ?

**Comment by merks** [comment]
I don't think we should not inflict what is all too often poorly tested, unreleased content...

**Comment by laeubi** [comment]
I think both sites can be added at laest  with disabled = true ... but actually sdk.product is more to produce the sdk.zip than something one would really use as an Eclipse-IDE directly?

**Comment by iloveeclipse** [comment]
> but actually sdk.product is more to produce the sdk.zip than something one would really use as an Eclipse-IDE directly?

I use plain SDK directly as IDE since years. What is wrong with that, why one wouldn't use that?

> Maybe its worth to even use https://download.eclipse.org/releases/milestone/ ?

We don't know who and how uses SDK. Assuming there are people that expect to see "stable" updates only, this is not what they would want for sure.

> That makes me wonder about perhaps using this instead:
> https://download.eclipse.org/releases/latest/

I think this could be the right way.
